audio/speech/getting-started/get_started_with_chirp_3.ipynb

```diff
@@ -336,6 +432,73 @@
       "source": [
         "display(Audio(response.audio_content))"
       ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "LYKj5gmDG7nT"
+      },
+      "source": [
+        "### Synthesize speech using streaming processing\n",
+        "\n",
+        "Chirp 3 HD voices also support streaming text-to-speech conversion using the `streaming_synthesize` method.  Unlike the standard `synthesize_speech`, which handles single requests, `streaming_synthesize` processes continuous streams of text, generating corresponding audio streams.\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "ll9rWo-dQ8Cx"
+      },
+      "outputs": [],
+      "source": [
+        "def synthesize_streaming(\n",
```

Comment:
Move into the helper functions cell. One of those methods needs it.

Reply:
done